### PR TITLE
panel: middle click to close window

### PR DESCRIPTION
**Pull request by carterprince**

### CODE REVIEW

File: src/panel/widgets/window-list/toplevel.cpp [hunk 1]
```diff
@@ -186,6 +186,10 @@ class WayfireToplevel::impl
         {
             menu.popup(event->button, event->time);
             return true; // It has been handled.
+        } else if ((event->type == GDK_BUTTON_PRESS) && (event->button == 2)) // middle click
```
Comment (soreau) [inline src/panel/widgets/window-list/toplevel.cpp:189]: This probably should only be enabled with an option that is disabled by default.

Comment (carterprince) [reply src/panel/widgets/window-list/toplevel.cpp]: Done!

File: metadata/panel.xml [hunk 2]
```diff
@@ -74,6 +74,10 @@
 		<_short>Background Color</_short>
 		<default>gtk_headerbar</default>
 	</option>
+	<option name="middle_click_close" type="bool">
```
Comment (soreau) [inline metadata/panel.xml:77]: This should be taken out of the `General` group and placed in a new `Window List` group.

**Review by ammen99 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**